OrbitCore/TracepointEventBuffer.cpp

```diff
@@ -8,23 +8,29 @@ using orbit_client_protos::TracepointEventInfo;
 
 void TracepointEventBuffer::AddTracepointEventAndMapToThreads(uint64_t time,
                                                               uint64_t tracepoint_hash,
-                                                              int32_t thread_id) {
+                                                              int32_t thread_id, int32_t process_id,
+                                                              bool is_same_pid_as_target) {
+  if (!is_same_pid_as_target) {
+    return;
```

Comment:
Maybe add a `TODO` here explaining that tracepoint events with `!is_same_pid_as_target` will also have to be stored when implementing the track showing tracepoint events from all processes in the system.